src/components/searchForm/searchForm.test.js

```diff
@@ -0,0 +1,32 @@
+import '@testing-library/jest-dom'
+
+import * as React from 'react'
+import { fireEvent, render, screen, debug } from '@testing-library/react'
+import SearchForm from './searchForm';
+const searchCB = jest.fn();
+const setup = (initialValue) => {
+    const utils = render(<SearchForm initialSearchTerm={initialValue} onChange={searchCB}></SearchForm>);
+    const searchInput = screen.getByLabelText('search-box');
+    return { ...utils, searchInput };
+}
+test('check if the search box is rendered with inital value', () => {
```

Comment:
Add all the test within a describe block.

Reply:
Wrapped tests inside `describe` block.